graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/execution/FunctionDataFetcher.kt

```diff
@@ -98,7 +98,7 @@ open class FunctionDataFetcher(
      *
      * This is currently achieved by using a Jackson [ObjectMapper].
      */
-    protected open fun convertParameterValue(param: KParameter, environment: DataFetchingEnvironment): Any? {
+    private fun convertParameterValue(param: KParameter, environment: DataFetchingEnvironment): Any? {
```

Comment:
by keeping it open we could allow others to implement some custom logic here, e.g. how to handle file uploads or skip converting the type if underlying object is already a correct type

Reply:
Did not think about that possibility. Let's discuss in more about both on the thread below.